components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -17,7 +17,7 @@
       <ng-container *ngIf="(sortedProjects$ | async)?.length > 0">
         <chef-toolbar>
           <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
-            <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
+            <chef-button [disabled]="(sortedProjects$ | async)?.length >= MAX_PROJECTS" id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
```

Comment:
@susanev this is a UX nitpick, but I think it's hard to immediately tell why the create button is disabled. I know it's in the sub-heading but my eye tends to skip over that. Some kind of "no more projects allowed" message to the right of the button would be helpful. (though I know there's supposed to be an `Update` Button there soon so I get it you don't want to mess with that design)

Reply:
I totally agree but unfortunately with ingest rules there are going to be other buttons to the right so we can't do that approach anymore. If we get user feedback on that from early customers we can find a better solution.

Reply:
Also, this limit should not be in place for more than a few months... hopefully.